### PR TITLE
Update codes part3

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: src/YearOverviewHeatmap/index.tsx [hunk 1]
```diff
@@ -177,12 +178,22 @@ export function YearOverviewHeatMap({
...
+      // Add text color
+      selectAll('.x-axis , .y-axis')
+        .selectAll('text')
+        .attr('fill', 'var(--primary_color)');
+
+      // Add path color
+      selectAll('.x-axis, .y-axis')
+        .selectAll('path')
+        .attr('stroke', 'var(--primary_color)');
+
```
Comment (maneetgoyal) [inline src/YearOverviewHeatmap/index.tsx:196]: Was the code copied from the private repo or was the change made line-by-line? Copying would be preferred in this case.

Comment (qur786) [reply src/YearOverviewHeatmap/index.tsx:196]: Yeah sir, I copied the code.

**Comment by qur786** [comment]
@maneetgoyal sir the private repo has some issue with label visibility on HSL and Default color mode, we should fix it before releasing a new version here.

**Review by maneetgoyal — APPROVED** [review]
Merge if the build is passing.

**Comment by maneetgoyal** [comment]
> @maneetgoyal sir the private repo has some issue with label visibility on HSL and Default color mode, we should fix it before releasing a new version here.

Cool

**Pull request merged**